tsdb/mapper_test.go

```diff
@@ -569,11 +565,53 @@ func openSelectMapperOrFail(t *testing.T, shard *tsdb.Shard, stmt *influxql.Sele
 func aggIntervalAsJson(t *testing.T, mapper *tsdb.SelectMapper) string {
 	r, err := mapper.NextChunk()
 	if err != nil {
-		t.Fatalf("failed to get chunk from aggregate mapper: %s", err.Error())
+		t.Fatalf("failed to get next chunk from aggregate mapper: %s", err.Error())
+	}
+	return mustMarshalMapperOutput(r)
+}
+
+// mustMarshalMapperOutput manually converts a mapper output to JSON, to avoid the
```

Comment:
Since production code overrides JSON-encoding, the test code now needs an explicit simpler version.